sbol_utilities/sbol3_genbank_conversion.py

```diff
@@ -213,53 +253,9 @@ def convert_genbank_to_sbol3(self, gb_file: str, sbol3_file: str = "sbol3.nt", n
             # Setting properties for GenBank's extraneous properties not settable in any SBOL3 field.
             self._store_extra_properties_in_sbol3(doc, comp, seq, record)
 
-            if record.features:
```

Comment:
The extraction of this code block to a separate method is really great, and a nice step towards #168